docs/rules/no-return-value-from-each-iteratee.md

```diff
@@ -0,0 +1,37 @@
+# No return value from ```_.each``` iteratee
+
+Many implementations of "each" allow for an early exit from the loop by returning a magic value like "false". Underscore does not offer an early exit. Furthermore, it does not do anything with the each's iteratee's return value.
+
+This rule would disallow returning anything from each's iteratee. It would be good for it to allow a return with no value, to allow people to short-circuit/exit early from the iteratee itself.
```

Comment:
Could be replaced by "This rule disallows returning anything from each's iteratee."